src/pybind/mgr/dashboard/frontend/src/app/ceph/new-dashboard/mon-summary.pipe.ts

```diff
@@ -0,0 +1,16 @@
+import { Pipe, PipeTransform } from '@angular/core';
+
+@Pipe({
+  name: 'monSummary'
+})
+export class MonSummaryPipe implements PipeTransform {
+  transform(value: any): any {
+    if (!value) {
+      return '';
+    }
+
+    const result = $localize`${value.monmap.mons.length.toString()}`;
```

Comment:
You're aldready creating a string, and the localization doesn't really do anything with a fully dynamic string (only if the string had some fixed/static parts).
```suggestion
    const result = `${value.monmap.mons.length}`;
```